src/VirtoCommerce.CatalogModule.Web/Scripts/widgets/dashboard.js

```diff
@@ -1,13 +1,13 @@
-﻿angular.module('virtoCommerce.catalogModule')
+angular.module('virtoCommerce.catalogModule')
 .controller('virtoCommerce.catalogModule.dashboard.catalogsWidgetController', ['$scope', '$state', 'virtoCommerce.catalogModule.catalogs', function ($scope, $state, catalogs) {
     $scope.data = { count: '', descr: 'Catalogs' };
 
     $scope.widgetAction = function () {
         $state.go('workspace.catalog');
     };
 
-    catalogs.getCatalogs({}, function (data) {
-        $scope.data.count = data.length;
+    catalogs.search({take:1000}, function (data) {
```

Comment:
Here I think we should specify take:0, as we only using length. Please check, possible redundant queries

Reply:
fixed